packages/story-editor/src/components/colorPicker/basicColorList.js

```diff
@@ -108,6 +109,9 @@ function BasicColorList({
   const colorType = isLocal
     ? __('local', 'web-stories')
     : __('global', 'web-stories');
+
+  const isBackground = changedStyle !== 'color';
```

Comment:
Wouldn't `changedStyle === 'background'` work here?

Reply:
(Since there are other `changedStyle` values as well possible, such as "border-color", would be better if we would check directly if the `changedStyle` is `background`.)